PacketDefinitions420/PacketNotifier.cs

```diff
@@ -1665,11 +1662,11 @@ public void NotifyNPC_CastSpellAns(INavigationGrid navGrid, ISpell s, Vector2 st
             };
             if (s.Level > 0)
             {
-                castInfo.ManaCost = s.SpellData.ManaCost[s.Level - 1];
+                castInfo.ManaCost = s.SpellData.ManaCost[s.Level];
             }
             else
             {
-                castInfo.ManaCost = s.SpellData.ManaCost[s.Level];
+                castInfo.ManaCost = s.SpellData.ManaCost[0];
```

Comment:
in the else case the level can be negative?

Reply:
If it is negative, yes. Unlikely, but just in case.